src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -714,12 +714,6 @@ def _gen_dynamic_list(function_config: Dict) -> List:
 
 
 def _generate_package_tools(keys: Optional[List[str]] = None) -> dict:
-    import imp
-
-    import pkg_resources
-
-    imp.reload(pkg_resources)
```

Comment:
who wrote this line previously?

Reply:
Looks like this PR also delete this: https://github.com/microsoft/promptflow/pull/1375/files#diff-47208ac35b30920275fcd5e55d662647ef360129359bdc77fddd2a2157b6f47e

Reply:
This code was wrote on ADO. Seems we can wait for the referred PR to get merged